src/sentry/templates/sentry/emails/email-styles.html

```diff
@@ -764,21 +768,68 @@
     text-transform: uppercase;
   }
 
-  table.alert-rule-table thead th {
+  .alert-badge {
+    display: inline-block;
+    border-radius: 3px;
+    text-transform: uppercase;
+    font-size: 12px;
+    color: #fff;
+    background: #C6BECF;
+    padding: 4px 8px;
+    margin-right: 12px;
+  }
+
+  .alert-info-container {
+    border-radius: 3px;
+    padding: 16px;
+  }
+
+  .alert-info-container table {
+    margin: 0;
+  }
+
+  .alert-info-container h4 {
+    margin: 0 0 12px;
+    display: flex;
```

Comment:
If flex isn't available this will what, not align center? That seems okay.

Reply:
Yeah, is flex available in any email client?

Reply:
https://www.caniemail.com/search/?s=flex Supports not bad, but I actually think this isn't as important as it once was since there's no longer text next to the badge.